FrejaEidClient/src/main/java/com/verisec/frejaeid/client/beans/general/OrganisationIdAttribute.java

```diff
@@ -0,0 +1,74 @@
+package com.verisec.frejaeid.client.beans.general;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+public class OrganisationIdAttribute {
+
+    private final String key;
+    private final String friendlyName;
+    private final String value;
+
+    /**
+     * Creates instance of {@linkplain OrganisationIdAttribute}.
+     *
+     * @param key            identifier of the attribute. It is not displayed to the end user.
+     *                       It cannot be {@code null} or empty.
+     *
+     * @param friendlyName   name of attribute that will be set to user (for
```

Comment:
**[MEDIUM]**
Length limits that we decided to use are:

key - 64
value - 256
displayText - 64

Reply:
Done